### PR TITLE
Update layers order

### PR DESCRIPTION
**Pull request by nnngrach**
[issue](https://github.com/osmandapp/OsmAnd-iOS/issues/2117)

Screenshot 2022-10-06 at 12 36 02

### CODE REVIEW

File: Sources/Controllers/Map/Layers/OATransportStopsLayer.mm [hunk 1]
```diff
@@ -139,7 +139,7 @@ - (void) doShowStopsOnMap
         if (_transportStopSymbolsProvider)
             [self.mapView removeTiledSymbolsProvider:_transportStopSymbolsProvider];
 
-        _transportStopSymbolsProvider.reset(new OsmAnd::TransportStopSymbolsProvider(self.app.resourcesManager->obfsCollection, self.baseOrder - 1000, _stopRoute.route, std::make_shared<OACoreResourcesTransportRouteIconProvider>(OsmAnd::getCoreResourcesProvider(), self.mapViewController.displayDensityFactor, 1.0)));
+        _transportStopSymbolsProvider.reset(new OsmAnd::TransportStopSymbolsProvider(self.app.resourcesManager->obfsCollection, [self getPointsOrder], _stopRoute.route, std::make_shared<OACoreResourcesTransportRouteIconProvider>(OsmAnd::getCoreResourcesProvider(), self.mapViewController.displayDensityFactor, 1.0)));
```
Comment (ivanPyrohivskyi) [inline Sources/Controllers/Map/Layers/OATransportStopsLayer.mm:142]: self getPointsOrder - 1000

File: Sources/Controllers/Map/Layers/OAMapLayer.h [hunk 2]
```diff
@@ -21,8 +21,11 @@
 @property (nonatomic, readonly) OAMapRendererView *mapView;
 @property (nonatomic, readonly) BOOL nightMode;
 @property (nonatomic, readonly) CGFloat displayDensityFactor;
+@property (nonatomic, readonly) int baseOrder;
```
Comment (alex-osm) [inline Sources/Controllers/Map/Layers/OAMapLayer.h:24]: define pointsOrder similar to baseOrder
`@property (nonatomic, readonly) int pointsOrder;`

File: Sources/Controllers/Map/Layers/OAMapLayer.h [hunk 3]
```diff
@@ -44,4 +47,7 @@
 
 - (BOOL) isVisible;
 
+- (int) getPointsOrder;
+- (void) setPointsOrder:(int)pointsOrder;
```
Comment (alex-osm) [inline Sources/Controllers/Map/Layers/OAMapLayer.h:51]: Drop these methods

File: Sources/Controllers/Map/Layers/OAMapLayer.mm [hunk 4]
```diff
@@ -23,6 +23,7 @@ @interface OAMapLayer()
 @implementation OAMapLayer
 {
     MBProgressHUD *_progressHUD;
+    int _pointsOrder;
```
Comment (alex-osm) [inline Sources/Controllers/Map/Layers/OAMapLayer.mm:26]: Not needed here

**Pull request merged**